install-php-extensions

```diff
@@ -2234,12 +2234,13 @@ installMicrosoftSqlServerODBC() {
 			;;
 		debian)
 			printf -- '- installing the Microsoft APT key\n'
-			if test $DISTRO_VERSION_NUMBER -eq 11; then
-				curl -sSLf -o /etc/apt/trusted.gpg.d/microsoft.asc https://packages.microsoft.com/keys/microsoft.asc
+			if test $DISTRO_VERSION_NUMBER -ge 13; then
```

Comment:
Can we please use https://packages.microsoft.com/debian/12/prod/pool/main/ source which works until 13/Trixies packages are officially added?

Currently this prevents us testing PHP 8.5 and there is no easy workaround.

Reply:
That is true, but still a lot of LoC would have to copied.

Reply:
Here are my experiments: https://github.com/atk4/data/commit/ecc7c3b0e79abc3dbf25515b9abec3115f07879f

I used the instruction from you above.

Then I tried to install https://github.com/stancl/msphpsql/commit/8aa519d9e389739997dd94a8e697ed.

But this installer expects probably the sources to be in the repo root? Is that true? Can I use #1113 and specify a subpath of the source like for this usecase (there are 2 extensions in the repo)?